lib/perl/Genome/Site/TGI/CleTest/prime-db.pl

```diff
@@ -143,38 +144,14 @@ sub load_users_by_username {
 }
 
 sub main {
-    my @instrument_data_ids = qw(
-        2893814999
-        2893815000
-        2893815001
-        2893815002
-        2893815003
-        2893815016
-        2893815018
-        2893815020
-        2893815023
-        2893815024
-        2893815447
-        2893815448
-        2893815449
-        2893815451
-        2893815484
-        2893815485
-        2893815489
-        2893815492
-    );
+
+    use Genome::Site::TGI::CleTest;
```

Comment:
This could be located up top with the other `use` statements, since all occur at "compile time".